packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -146,6 +162,14 @@ class Auth {
         DATA_MAP[name].callback?.()
     }
 
+    private clearStorage() {
+        Object.keys(DATA_MAP).forEach((key) => {
+            DATA_MAP[key as keyof typeof DATA_MAP].storage.delete(
```

Comment:
Nit: Can we create a type for here? It can probably be used in some other places